examples/language-modeling/run_lora_clm.py

```diff
@@ -46,7 +47,7 @@
 
 from optimum.habana import GaudiConfig, GaudiTrainer, GaudiTrainingArguments
 from optimum.habana.utils import set_seed
-
+from optimum.habana.peft.layer import GaudiLoraLayerLinearforward
```

Comment:
Should we do that everytime we use FSDP and LoRA? Or is it necessary for this example only?
If it's always necessary, it's better to do that in `optimum.habana.transformers.modeling_utils.py`.

Reply:
I think it's okay yes. Maybe LoRA inference in the text-generation example could be impacted, I'll check that.

Reply:
@regisss My understanding was that LoRA is used only for finetuning. But please check and resolve this conversation when you get a chance. Thanks.